Makefile

```diff
@@ -25,14 +25,27 @@ GIT_COMMIT := $(shell git rev-parse HEAD 2> /dev/null || true)
 
 all: binary
 
-binary:
+binary: ${DEST}skopeo
+
+# Build a docker image (skopeobuild) that has everything we need to build.
+# Then do the build and the output (skopeo) shold appear in ${DEST} dir (or .).
+${DEST}skopeo: cmd/skopeo
+	docker build ${DOCKER_BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
+	-docker rm -f skopeobuild
+	docker run -v ${PWD}:/src/github.com/projectatomic/skopeo \
+		--name skopeobuild --env DEST=/ skopeobuildimage make binary-local
+	docker cp skopeobuild:/skopeo ${DEST}skopeo
```

Comment:
This `DEST` business feels awkward, I’d rather remove it than spread all over the place.  @runcom ?

- The rules above don’t depend on `DEST` being supported: the `docker cp` could just as well do 
```sh
docker cp skopeobuild:/src/github.com/projectatomic/skopeo/skopeo ….
```
- The caller needs to add a trailing `/` to `DEST`, which is unintuitive.
- `DEST` could be easily confused with the more relevant `DESTDIR`.  (I am interested in this because the policy stuff will make usage of `make install` more important; installing the binary without the policy could, depending on what we decide for the compiled-in defaults, result in a broken application.)